apps/ledger-live-desktop/src/renderer/families/celo/WithdrawFlowModal/types.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import type { TFunction } from "react-i18next";
+import type { Device } from "@ledgerhq/live-common/lib/hw/actions/types";
+import type { Step } from "~/renderer/components/Stepper";
+
+import type { Account, TransactionStatus, Operation } from "@ledgerhq/live-common/lib/types";
+
+import type { Transaction } from "@ledgerhq/live-common/lib/families/tron/types";
+
+export type StepId = "amount" | "connectDevice" | "confirmation";
+
+export type StepProps = {
+  t: TFunction,
+  transitionTo: string => void,
```

Comment:
One thing I think would be helpful here is if we set the parameter type to `StepId`.  Something like:

```
transitionTo: (string: StepId) => void,
```

That way, we won't have to worry about misusing this function.  Could we make a similar change in all of the `[XXX]FlowModal/types.js` files too?